packages/agents/router/src/lib/operations/execute.ts

```diff
@@ -108,7 +115,8 @@ export const execute = async (params: OriginTransfer): Promise<void> => {
 
   logger.debug("Sanity checks passed", requestContext, methodContext, { liquidity: balance.toString() });
 
-  const fee = DEFAULT_ROUTER_FEE;
+  // TODO. Needs to calculate fee amount in BigNumber and pass it into Bid.
+  const fee = "0";
```

Comment:
This 'fee' can actually be removed from bids for now. It was a placeholder, as I initially thought routers would be sending competitive custom fees/rates sooner.

But this will actually stay hardcoded on-chain for now. The `LIQUIDITY_FEE_NUMERATOR` and `LIQUIDITY_FEE_DENOMINATOR` are what's used to determine the LP fee the router actually gets (approximately).

Reply:
Or at least change the TODO here pls

Reply:
I don't think so. I think the fee should be there for off-chain calculation even though we use hardcoded values rn.

Reply:
any off-chain calculations should pull the current values from on-chain state as needed.

This value is 0.05% everywhere on-chain rn, and it's enforced on-chain. Nothing we record here is relevant.

In the future, we may allow routers to bid their own percentages (e.g. if a router wants to be competitive, they might bid 0.04% if it were an option). But that feature won't exist for a while. It's safe to remove `fee` from the `Bid` type for now.